lib/model/Repository.js

```diff
@@ -102,6 +102,17 @@ function Repository(serviceUrl) {
         objects.getObjectByID(self.token, type, id, callback);
     };
 
+    /**
+     * See [getToken in Tokens]{@link Tokens#getToken}. A token object will be returned that is associated with the given {@link apiToken}
+     * 
+     * @param {?string} tenantName 
+     * @param {string} apiToken 
+     * @param {function} callback 
+     */
+    this.getToken = function(tenantName, apiToken, callback) {
```

Comment:
@cratter49 Does this really need to accept any arbitrary token, or can it use self.token instead?

Reply:
That's true. The token class variable should have been set if either of the `Repository` connect methods were called.